vindaloo/objects.py

```diff
@@ -1,6 +1,17 @@
 import copy
 from typing import Optional
 
+__all__ = (
+    'Dict',
+    'List',
```

Comment:
Nebude se yo bit s typingem? :-)

Reply:
To jsme řešili a nebude, protože v k8s konfiguraci typing nepoužíváme. Pokud by ho někdo chtěl, může si to aliasovat :)